### ISSUE TITLE
Keyboard access to `Add Reference` dialog

### ISSUE THREAD

**Issue by Wall-AF** [NationalSecurityAgency/ghidra#5761]
**Describe the bug**
You cannot use the keyboard to navigate the `Add Reference` dialog to alter the `Type of Reference`!

**To Reproduce**
1. Pick any application to RE
2. Choose `Add Reference from...` on the `References` menu from the `Listing:`Panel
3. Try tabbing etc around the dialog
4. See you cannot get anywhere other than the 2 input areas `Register`&`Ref-Type`

**Expected behavior**
To be able to navigate **ALL** dialogs without having to resort to the mouse!

This isn't the only remaining dialog with keyboard navigation issues!

**Screenshots**
![image](https://github.com/NationalSecurityAgency/ghidra/assets/67698927/6e7aa20a-bd5c-4f61-8a26-94fdd558f58b)

**Attachments**
N/A

**Environment (please complete the following information):**
 - OS: Windows 11
 - Java Version: 17.0,3.1
 - Ghidra Version: 10.4.DEV
 - Ghidra Origin: locally built from 26d4bd910

**Additional context**
N/A
